platform/ios/MGLMapView.mm

```diff
@@ -2710,23 +2707,10 @@ void notify() override
         // no-op
     }
 
-    void notifyMapChange(mbgl::MapChange change, std::chrono::steady_clock::duration delay = std::chrono::steady_clock::duration::zero()) override
+    void notifyMapChange(mbgl::MapChange change) override
     {
-        if (delay != std::chrono::steady_clock::duration::zero())
-        {
-            dispatch_after(dispatch_time(DISPATCH_TIME_NOW, std::chrono::duration_cast<std::chrono::nanoseconds>(delay).count()), dispatch_get_main_queue(), ^
-            {
-                [nativeView performSelector:@selector(notifyMapChange:)
-                                 withObject:@(change)
-                                 afterDelay:0];
-            });
-        }
-        else
-        {
-            assert([[NSThread currentThread] isMainThread]);
-
-            [nativeView notifyMapChange:@(change)];
-        }
+        assert([[NSThread currentThread] isMainThread]);
```

Comment:
On second thought, that won’t work because `_cmd` is only available inside Objective-C classes. `NSCAssert()` would still be preferable to `assert()` in Objective-C code, though.